API/README.md

```diff
@@ -1,27 +1,29 @@
-# NCATS Translator Reasoners API
+# NCATS Translator Graph API
```

Comment:
Looks like you have opinions about #115 😄.

Reply:
ha ha, you caught me! Just checking to see if anyone is reading these. I don't actually have a string opinion. It seemed like Reasoners was out, so I thought I'd try what seemed like the best suggestion I've seen on for size.

Reply:
I reverted the name change. How about everything else? Approvals for the rest?